### ISSUE TITLE
Accents are not allowed in character extensions

### ISSUE THREAD

**Issue by gamaliny** [piersdeseilligny/betterfountain#58]
Hello, 
I'm writting in French and I cannot add acute accent or any other accents in the parenthetical or in the character's name.

**Comment by piersdeseilligny** [comment]
@gamaliny could you give an example of a character name and/or parenthetical that isn't recognized?

Normally this should work, as such:
![image](https://user-images.githubusercontent.com/42496508/81008836-7ed80880-8e4b-11ea-945d-7e12d9849c47.png)

**Comment by gamaliny** [comment]
![image](https://user-images.githubusercontent.com/33547685/81009670-5b19c000-8e55-11ea-9aac-000e1f94eaff.png)
It's strange, I thought it was the accent which bugged the preview (and the export) conversion.

**Comment by gamaliny** [comment]
I tried the same example that you quote but I still have the same parsing error : 
![image](https://user-images.githubusercontent.com/33547685/81010182-222e1b00-8e56-11ea-8ad7-989a285a4e89.png)
I'm using 1.4.5. Better Fountain version and Visual Code 1.43.2.

**Comment by piersdeseilligny** [comment]
Ahh I see. What you're referring to are _character extensions_, rather than _parentheticals_.

Normally, a **parenthetical** (which specifies the delivery of the line) is on the line below the character name, for example:
```
ÉDOUARD
(très fort)
Bonjour.
```

This differs from **character extensions**, which are technical descriptions about the placement of the character, such as `(O.S.)` which means the line is being said off-screen, or `(V.O.)`, which means it is a voice-over.
```
ÉDOUARD (O.S.)
(très fort)
Bonjour
```


Basically this is a formatting issue: what you're writing as a character extension (on the same line as the character name) should be a parenthetical (on the line below the character name).

I guess character extensions could accept accents, but they are a lot less variable than parentheticals; it's generally `(V.O.)`, `(O.S.)`, or something similar, written in all caps.

**Comment by gamaliny** [comment]
> This differs from character extensions, which are technical descriptions about the placement of the character, such as (O.S.) which means the line is being said off-screen, or (V.O.), which means it is a voice-over.
All right I think I mixed-up the two thanks for the precision !
My problem is solved I was supposed to use parenthetical and I wrongly used the character extensions instead.
